### ISSUE TITLE
Extend request-pattern schema with currently not specified properties

### ISSUE THREAD

**Issue by picimako** [wiremock/wiremock#2178]
### Proposal

Based on `com.github.tomakehurst.wiremock.matching.RequestPattern` there are a couple of JSON properties that are available to be specified in a JSON stub mapping, but are not included in the `request-pattern.yaml` schema.

The ones I see that could be included are: `scheme`, `host`, `port`, `customMatcher`, `multipartPatterns`.

I'm planning on including them in the WireMocha IntelliJ plugin as well, but first, I want to make sure that they are correctly specified, and that they would be in sync with the schema in WireMock.

The schema update could look something like this:

```
...
    properties:
      ...
      scheme:
        type: string
        description: The URI scheme
      host:
        type: object
        description: 'URI host pattern to match against in the "<predicate>": "<value>" form'
      port:
        type: integer
        description: The HTTP port number
      customMatcher:
        type: object
        description: Custom request matcher to match against
        properties:
          name:
            type: string
            description: The matcher's name specified in the implementation of the matcher.
          parameters:
            type: object
      multipartPatterns:
        type: array
        description: Multipart patterns to match against headers and body
        items:
          type: object
          properties:
            name:
              type: string
            matchingType:
              type: string
              default: ANY
              enum:
                - ALL
                - ANY
            headers:
              type: object
            bodyPatterns:
              type: array
              items:
                type: object
```

**Comment by oleg-nenashev** [comment]
No objections from my side. WDYT @tomakehurst ?

**Comment by picimako** [comment]
Closing as this has already been released in 3.0.0-beta-12.
